packages/framework/collabspaces/src/test/collabSpaces.spec.ts

```diff
@@ -494,12 +593,14 @@ describe("Temporal Collab Spaces", () => {
 		// such conversion works correctly.
 	}
 
-	it("Channel overwrite with syncronization", async () => {
-		await ChannelOverwrite(true);
-	});
+	describe("Channel Overwrite", () => {
```

Comment:
You are a bit behind on changes - please merge latest changes (there are more tests and more arguments here